arraySort.js

```diff
@@ -0,0 +1,22 @@
+function arraySort(a, b) {
+    return b - a
+    //return a<b?1 : a>b?-1 : 0
+}
+
+var x = [1, 4, 7, "x", 3, 5, 8, "t", 0, 6, 10, "a"];
+x.sort(arraySort)
+x.reverse()
+i = 0
+while (i < x.length) {
+    if (isNaN(x[i])) {
+        characters.push(x[i]);
+    }
+    if (x[i] % 2 == 0) {
+        evenNumbers.push(x[i]);
+    }
+    if (x[i] % 2 == 1) {
+        oddNumbers.push(x[i]);
+    }
+    i++;
+}
+console.log(props)
```

Comment:
Why are you logging props here, Where do you get the props variable from?

Reply:
@dennisja my object was mistakenly cut before i commited. The props is supposed to be up. I've added the actual code on the second commit